.nvmrc

```diff
@@ -0,0 +1 @@
+v22.15.1
```

Comment:
If you're using mise, would it be better to say v22 instead of specifiying today's patch version?

The trouble with this approach is that this is arguably worse than what was there before. Previously, GitHub were keeping this up to date for us and if there was a problem we'd (likely) find out at build time.

`22.15.1` is a good choice today but we have no tooling to keep this up to date. Unless we remember to regularly bump this version number, it'll start accumulating vulnerabilities.